app/src/main/java/com/nextcloud/talk/models/json/chat/ChatMessage.java

```diff
@@ -251,6 +252,245 @@ public String getSystemMessage() {
+    }
+
+    public String getActorType() {
+        return this.actorType;
+    }
+
+    public String getActorId() {
+        return this.actorId;
+    }
+
+    public String getActorDisplayName() {
+        return this.actorDisplayName;
+    }
+
+    public long getTimestamp() {
+        return this.timestamp;
+    }
+
+    public String getMessage() {
+        return this.message;
+    }
+
+    public HashMap<String, HashMap<String, String>> getMessageParameters() {
+        return this.messageParameters;
+    }
+
+    public SystemMessageType getSystemMessageType() {
+        return this.systemMessageType;
+    }
+
+    public boolean isReplyable() {
+        return this.replyable;
+    }
+
+    public ChatMessage getParentMessage() {
+        return this.parentMessage;
+    }
+
+    public Enum<ReadStatus> getReadStatus() {
+        return this.readStatus;
+    }
+
+    public List<MessageType> getMessageTypesToIgnore() {
+        return this.messageTypesToIgnore;
+    }
+
+    public void setGrouped(boolean isGrouped) {
+        this.isGrouped = isGrouped;
+    }
+
+    public void setOneToOneConversation(boolean isOneToOneConversation) {
+        this.isOneToOneConversation = isOneToOneConversation;
+    }
+
+    public void setActiveUser(UserEntity activeUser) {
+        this.activeUser = activeUser;
+    }
+
+    public void setLinkPreviewAllowed(boolean isLinkPreviewAllowed) {
+        this.isLinkPreviewAllowed = isLinkPreviewAllowed;
+    }
+
+    public void setJsonMessageId(int jsonMessageId) {
+        this.jsonMessageId = jsonMessageId;
+    }
+
+    public void setToken(String token) {
+        this.token = token;
+    }
+
+    public void setActorType(String actorType) {
+        this.actorType = actorType;
+    }
+
+    public void setActorId(String actorId) {
+        this.actorId = actorId;
+    }
+
+    public void setActorDisplayName(String actorDisplayName) {
+        this.actorDisplayName = actorDisplayName;
+    }
+
+    public void setTimestamp(long timestamp) {
+        this.timestamp = timestamp;
+    }
+
+    public void setMessage(String message) {
+        this.message = message;
+    }
+
+    public void setMessageParameters(HashMap<String, HashMap<String, String>> messageParameters) {
+        this.messageParameters = messageParameters;
+    }
+
+    public void setSystemMessageType(SystemMessageType systemMessageType) {
+        this.systemMessageType = systemMessageType;
+    }
+
+    public void setReplyable(boolean replyable) {
+        this.replyable = replyable;
+    }
+
+    public void setParentMessage(ChatMessage parentMessage) {
+        this.parentMessage = parentMessage;
+    }
+
+    public void setReadStatus(Enum<ReadStatus> readStatus) {
+        this.readStatus = readStatus;
+    }
+
+    public void setMessageTypesToIgnore(List<MessageType> messageTypesToIgnore) {
+        this.messageTypesToIgnore = messageTypesToIgnore;
+    }
+
+    public boolean equals(final Object o) {
```

Comment:
creepy... ;) i think delombok should be the last emergency solution...

Reply:
Yes :/ 
Maybe it is better to move those classes one by one to Kotlin data classes…